Sources/View/NibLoadable.swift

```diff
@@ -28,6 +31,11 @@ public extension NibLoadable {
   static var nib: UINib {
     return UINib(nibName: String(describing: self), bundle: Bundle(for: self))
```

Comment:
I'd then re-implement this in terms of `nibName`

Reply:
Right, revised.